tests/e2e_tests/helpers.py

```diff
@@ -0,0 +1,65 @@
+import os
+import re
+import secrets
+import time
+
+import requests
+from notifications_python_client import NotificationsAPIClient
+from playwright._impl._errors import Error as PlaywrightError
+from playwright.sync_api import Page
+
+
+def login_via_magic_link(page: Page, test_name: str, email_domain: str = "levellingup.gov.test"):
+    response = requests.get("http://authenticator.levellingup.gov.localhost:4004/magic-links")
```

Comment:
As we repeat `http://authenticator.levellingup.gov.localhost:4004/magic-links` three times in this method, maybe best to take out as a variable?

Reply:
I'm semi-intentionally leaving this as "ugly"/"inefficient" so that when we get around to [this ticket](https://dluhcdigital.atlassian.net/browse/FPASF-471) it's very obvious we need to do something to fix it properly.

Happy to do a halfway step now if you prefer though.